__tests__/app.test.js

```diff
@@ -353,4 +353,132 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("GET /api/articles (queries)", () => {
+        test("200 will returned and list of articles when all queries are provided", () => {
+            return request(app)
+                .get("/api/articles?topic=mitch&sort_by=title&order=asc")
+                .expect(200)
+                .then(({ body }) => {
+                    const articles = body.articles;
+                    expect(articles).toBeSortedBy("title", {
+                        descending: false,
+                    });
+                    expect(articles).toHaveLength(12);
+                    expect(articles.body).toBeUndefined();
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author", expect.any(String));
+                        expect(article).toHaveProperty("title", expect.any(String));
+                        expect(article).toHaveProperty("article_id", expect.any(Number));
+                        expect(article).toHaveProperty("topic", expect.any(String));
```

Comment:
Rather than expecting any string here, what topic should each of these articles have? We've queried with a topic of "mitch", but haven't actually checked it works here.